service-data-access/src/test/resources/test.json

```diff
@@ -0,0 +1,239 @@
+{
```

Comment:
Can you give this a better name? test.json is too generic. maybe suggests what it is or what it is testing. test-bundle-01?

Reply:
Renamed.